test/cmdlineTests.sh

```diff
@@ -71,6 +71,15 @@ function update_expectation {
     local newExpectation="${1}"
     local expectationFile="${2}"
 
+    if [[ $newExpectation == "" || $newExpectation -eq 0 && $expectationFile == "*/exit" ]]
```

Comment:
I think that globbing won't work inside a quoted string.
```suggestion
    if [[ $newExpectation == "" || $newExpectation -eq 0 && $expectationFile == */exit ]]
```